changelog.md

```diff
@@ -1,5 +1,9 @@
 # illogical changelog
 
+## 1.4.3
+
+- Prevent unexpected parsing of logical or comparison expression used as an operand in a comparison expression

```

Comment:
Consider changing to:

> Prevent unexpected parsing of any expression used as an operand in a comparison expression

I was originally focused on the significance of **logical or comparison expression** instead of **expression used as an operand in a comparison expression** and so emphasizing the latter may make this easier to understand.

Reply:
cc19350d8be214867a4d11ed4dfc8737527c8194